README.md

```diff
@@ -1,83 +1,39 @@
-Bugsnag Middleware for Silex
-==
+# Bugsnag Notifier for Silex
 
-The Bugsnag middleware for Silex integrates into Silex PHP applications.
+[![Build Status](https://img.shields.io/travis/bugsnag/bugsnag-silex/master.svg?style=flat-square)](https://travis-ci.org/bugsnag/bugsnag-silex)
```

Comment:
Add a docs link here, similar to the [badge on js](https://github.com/bugsnag/bugsnag-js/blob/master/README.md)